index.html

```diff
@@ -12,74 +12,130 @@
+              ><img src="images/Icon - Linkedin.png" alt="LinkedIn"
+            /></a>
           </li>
           <li>
-            <a href="#"><img src="images/Icon - Angelist.png" alt="Angelist" /></a>
+            <a href="#"
+              ><img src="images/Icon - Angelist.png" alt="Angelist"
+            /></a>
           </li>
           <li>
-            <a href="#"><img src="images/Icon - Twitter.png" alt="Twitter" /></a>
+            <a href="#"
+              ><img src="images/Icon - Twitter.png" alt="Twitter"
+            /></a>
           </li>
           <li>
-            <a href="#"><img src="images/Icon - Medium.png" alt="Medium" /></a>
+            <a href="#"
+              ><img src="images/Icon - Medium.png" alt="Medium"
+            /></a>
           </li>
         </ul>
-      </section>
+      </div>
+    </div>
   </div>
 
 
-  <section class="works">
-    <div class="grid-container">
+<section class="works"> 
+  <div class="grid-container">
+    
+      
+    <div class="second-desktop">
+      <h2 class="recents">My Recent Works</h2>
+      <div class="line"></div>
+    </div>
+       
 
+      
+        <div class="item1">
+            <img src="images/Img Placeholder.png" alt="placeholder" class="work-img">
 
-      <div class="item0">
-        <h2 class="recents">My Recent Works</h2>
-        <img src="images/Img Placeholder.png" alt="placeholder">
-      </div>
-
+            <div class="third-desktop">
+            <h2>Multi-Post Stories</h2>
+            <p class="stories">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                  <div class="lang list">
+            <ul>
+              <li><button>css</button></li>
+              <li><button>html</button></li>
+              <button>bootstrap</button>
+              <li><button>Ruby</button></li>
+            </ul>
+                  </div>
+                  <button class="see-project">See Project</button>
+          </div>
+        </div>
+      
 
 
-      <div class=" item1">
-        <h2>Multi-Post Stories</h2>
-        <p class="stories">A daily selection of privately personalized reads; no accounts or sign-ups required. has been
-          the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
-        </p>
-        <div class="lang list">
+      <div class="grid item2 mobile">
+        <h2>Project Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard</p>
+        <div class="lang">
           <ul>
-            <li><button>css</button></li>
             <li><button>html</button></li>
             <li><button>bootstrap</button></li>
             <li><button>Ruby</button></li>
           </ul>
+
         </div>
```

Comment:
I appreciate your effort in implementing the hovered version of the cards, but it's not active by default and it looks like this when the user hovers over the card. So please make sure to use a normal card layout here, and change it's style when the user hovers over the card.

![image](https://user-images.githubusercontent.com/34779928/206298264-acc78ac3-7810-42f4-b6d7-041239e59bd6.png)